### PR TITLE
dev, v3.0: add a view doc

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? 

Add a doc for TiDB views

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1609

**Note:** Changes about the SQL mode in the above PR will be updated by @TomShawn in his PR (that aligns docs with https://github.com/pingcap/docs-cn/pull/1495) later.

### Which version does your change affect? 

dev, v3.0

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/reference/sql/view.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: View
+summary: Learn how to use views in TiDB.
+category: reference
+---
+
+# View
+
+TiDB supports views. A view acts as a virtual table and its table schema is defined by the `SELECT` statement when you create a view. Using views has the following benefits:
```
Comment (dcalvin) [inline dev/reference/sql/view.md]:
```suggestion
TiDB supports views. A view acts as a virtual table and its table schema is defined by the `SELECT` statement when you create the view. Using views has the following benefits:
```

File: dev/reference/sql/view.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: View
+summary: Learn how to use views in TiDB.
+category: reference
+---
+
+# View
```
Comment (dcalvin) [inline dev/reference/sql/view.md]: I'd go with the plural form.

File: dev/reference/sql/view.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+TiDB supports views. A view acts as a virtual table and its table schema is defined by the `SELECT` statement when you create a view. Using views has the following benefits:
+
+- Exposing only safe fields and data to users to ensure the security of sensitive fields and data stored in the underlying table
+- Defining complex queries that frequently appear as views to make complex queries easier and more convenient
+
+## Query views
+
+Querying a view is similar to querying an ordinary table. However, when TiDB executes a view query, it expands this view into the `SELECT` statement defined when the view was created, and then executes the expanded query statement.
```
[the remark this answers is not in the document]
Comment (dcalvin) [reply dev/reference/sql/view.md]: A rephrasing for brevity.

File: dev/reference/sql/view.md (hunk 2 above)
Comment (lilin90) [reply dev/reference/sql/view.md]: Haha, originally, I used `views`. Then I changed it after taking a look at the TOC.

**Comment by lilin90** [comment]
@dcalvin Thanks! Comments addressed now. PTAL again~
@XuHuaiyu PTAL

**Pull request merged**